scalableminds/webknossos
# Warn if outside of bounding box

This PR adds a warning to the position view in tracings if the position is out of bounds.
Additionally this PR resolves issue #4336 by adding the error message to the ignored list.
The bug from #4472 is also fixed here.

### URL of deployed dev instance (used for testing):
- https://warnifoutsideofbbox.webknossos.xyz

### Steps to test:
- Open a dataset in view mode and move outside the dataset bounding box. The position view should turn orange and a tooltip should indicate that the user is outside the bounding box. 
   - also check arbitrary views: Here the rotation view should also turn orange if the user is outside of the bounding box.
- Repeat the same steps while doing any tracing task with a lower bounding box that the bounding box of the dataset. Once you leave the task bounding box the warning should appear again.

### Issues:
- fixes #4529
- fixes #4336
- fixes #4472

------
